src/content/guides/hot-module-replacement.md

```diff
@@ -191,7 +192,7 @@ __index.js__
   }
 
 - document.body.appendChild(component());
-+ let element = component(); // Store the element to re-render on print.js changes
++ const element = component(); // Store the element to re-render on print.js changes
```

Comment:
You cannot reassign a const, this will throw when we do `element = component()` later.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const